tests/snuba/api/endpoints/test_organization_stats_v2.py

```diff
@@ -184,14 +184,14 @@ def test_unknown_groupby(self):
         response = self.do_request(
             {
                 "field": ["sum(quantity)"],
-                "groupBy": ["cattygory"],
+                "groupBy": ["category_"],
```

Comment:
I'm assuming it doesn't matter as long as it isn't `category`, and hence offering an alternative.